lib/xnvme_be_spdk_dev.c

```diff
@@ -217,8 +217,7 @@ _spdk_env_init(struct spdk_env_opts *opts)
 
 	if (g_xnvme_be_spdk_env_is_initialized) {
 		XNVME_DEBUG("INFO: already initialized");
-		err = 0;
-		goto exit;
+		return 0;
```

Comment:
This is probably a matter of taste whether it is simpler/cleaner.
I found that it was simpler to be able to always look for:
```
INFO: spdk_env_is_initialized: 0/1
```
In the output, to figure out whether initialization succeeded, rather than ``spdk_env_is_initialized`` sometimes being there and sometimes not.

Also that the debug would explicitly show that ``INFO: already initialized`` when re-using.

Reply:
The commit has been dropped, I see what you mean